core/src/main/java/com/graphhopper/routing/InstructionsFromEdges.java

```diff
@@ -508,7 +508,7 @@ private void updatePointsAndInstruction(EdgeIteratorState edge, PointList pl) {
         }
         double newDist = edge.getDistance();
         prevInstruction.setDistance(newDist + prevInstruction.getDistance());
-        // todonow: why do we not account for turn times here ?
+        // todo: why do we not account for turn times here ?
```

Comment:
Should we create an issue as reminder for this?

Reply:
#1867